lib/views/teach_bpmf_view.dart

```diff
@@ -0,0 +1,421 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart' show rootBundle;
+import 'package:flutter_tts/flutter_tts.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:ltrc/data/models/word_model.dart';
+import 'package:ltrc/data/models/unit_model.dart';
+import 'package:ltrc/data/providers/word_provider.dart';
+import 'package:ltrc/extensions.dart';
+import 'package:ltrc/widgets/teach_word/tab_bar_view.dart';
+import 'package:ltrc/widgets/teach_word/stroke_order_animator.dart';
+import 'package:ltrc/widgets/teach_word/stroke_order_animation_controller.dart';
+import 'package:provider/provider.dart';
+
+class TeachBopomoView extends StatefulWidget {
+  final String char;
+  final int unitId;
+  final String unitTitle;
+  final bool isBpmf;
```

Comment:
Why do we need isBpmf here? I guess all the word cards on this page are all ㄅㄆㄇㄈ.

Reply:
I will combine the two teachWordView later, isBpmf is needed.